diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -50,6 +53,21 @@ open class DiktatJavaExecTaskBase @Inject constructor(
         } else {
             main = "com.pinterest.ktlint.Main"
         }
+        diktatExtension.reporter = when(diktatExtension.reporterType) {
```

Comment:
Actually, I think it should be passed to ktlint via CLI argument like `--reporter=json`. Adding classpath dependency might even be not needed (depending on whether they already include all reporters in `ktlint.jar`